utils/logger.js

```diff
@@ -0,0 +1,27 @@
+const pino = require('pino');
+
+require('../src/config');
+
+let logger;
+
+/* If in Development mode:
+ * Output logs to console (which is by default)
+ * Enable prettyPrint option and translate time from epoch time to local time.
+ * Set log level to LOG_LEVEL environment variable with 'debug' as default level.
+ */
+if (process.env.NODE_ENV === 'development') {
+  logger = pino({
+    prettyPrint: { translateTime: 'SYS: yyyy-mm-dd HH:MM:ss.l ' },
+    level: process.env.LOG_LEVEL || 'debug',
+  });
+} else if (process.env.LOG_FILE !== undefined) {
+  /* If in Production mode:
+   * Write logs to a specified path.
+   * Set log level to LOG_LEVEL environment variable with 'info' as default level.
+   */
+  logger = pino({ level: process.env.LOG_LEVEL || 'info' }, pino.destination(process.env.LOG_FILE));
```

Comment:
Need to check if `process.env.LOG_FILE` exists before you try and use this, it might not.

Reply:
The code checks whether the LOG_FILE variable is undefined or not. If it's not undefined, then it will set the pino.destibtion(..). See code below:
```javascript
else if (process.env.LOG_FILE !== undefined) {
  /* If in Production mode:
   * Write logs to a specified path.
   * Set log level to LOG_LEVEL environment variable with 'info' as default level.
   */
  logger = pino({ level: process.env.LOG_LEVEL || 'info' }, pino.destination(process.env.LOG_FILE));
}
```